hs_core/templates/pages/create-resource.html

```diff
@@ -55,7 +55,7 @@
                 <option value="SWATModelInstanceResource">SWAT Model Instance</option>
                 <option value="NetcdfResource">Multidimensional (NetCDF)</option>
                 <option value="TimeSeriesResource">Time Series</option>
-{#                <option value="ToolResource">Application</option>#}
+                <option value="ToolResource">Web App</option>

```

Comment:
I just looked through all of Brian's comments on the web app PR. They all
look reasonable. Though let's keep the name "Web App". This will help users
understand that it is a resource that links to another web based
application. It could link to HydroShare web apps or other ones. - Dan

On Mon, Dec 7, 2015 at 8:07 PM Brian Miles <notifications@github.com> wrote:

> In hs_core/templates/pages/create-resource.html
> <https://github.com/hydroshare/hydroshare/pull/772#discussion_r46910620>:
>
> > @@ -55,7 +55,7 @@
> >                  <option value="SWATModelInstanceResource">SWAT Model Instance</option>
> >                  <option value="NetcdfResource">Multidimensional (NetCDF)</option>
> >                  <option value="TimeSeriesResource">Time Series</option>
> > -{#                <option value="ToolResource">Application</option>#}
> > +                <option value="ToolResource">Web App</option>
>
> What about "HydroShare App"? Web app is a too generic and may be confusing
> to some users (HydroShare is after all a web app itself).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hydroshare/hydroshare/pull/772/files#r46910620>.
>